Content.Server/Traitor/Uplink/UplinkSystem.cs

```diff
@@ -22,8 +25,8 @@ public sealed class UplinkSystem : EntitySystem
     [Dependency] private readonly SharedSubdermalImplantSystem _subdermalImplant = default!;
     [Dependency] private readonly SharedMindSystem _mind = default!;
 
-    [ValidatePrototypeId<CurrencyPrototype>]
-    public const string TelecrystalCurrencyPrototype = "Telecrystal";
+    public static readonly EntProtoId TraitorUplinkStore = "StorePresetRemoteUplink";
```

Comment:
```suggestion
    public static readonly EntProtoId<StoreComponent> TraitorUplinkStore = "StorePresetRemoteUplink";
```

seems like a logical thing to attach